Stages.md

```diff
@@ -0,0 +1,138 @@
+    * Positive support to advance a stage does **not** imply:
+        * Commitment to eventually support a subsequent stage
+        * That the particular solution approach at the time of the signal or support will be the one eventually standardized (unless the advancement is to the Standardized stage)
+    * A negative response to advancing a stage does **not** imply:
+        * That the feature cannot continue in its current stage
+        * That a future request for support to advance a stage with a revised solution won’t succeed
+        * That all browsers will delay shipping the feature
+        * That the feature cannot continue to be incubated outside of the WHATWG
+    * A proposal can advance directly to later stages without going through the earlier stages, if it's already mature enough and has sufficient support to do so.
+* **Regressing stages**
+    * The browser engine representatives may review proposals and decide to regress stages—for example, if implementer support has been retracted for a particular solution (regressing from stage 3 to stage 2), or even if support has been retracted for solving the problem (i.e. a regression from stage 2 to stage 1). This situation is expected to be unusual, and should come with a very strong justification.
+
+<table>
+  <tr>
+   <td>
+Stage Name
+   </td>
+   <td>Entrance criteria
+   </td>
+   <td>This stage signifies
+   </td>
+   <td>Spec quality at entrance
+   </td>
+  </tr>
+  <tr>
+   <td>Stage 0
+<p>
+(Proposal)
+   </td>
+   <td>(None)
+   </td>
+   <td>That the proposer intends to use the stage process to work on their idea, with the goal of it landing in WHATWG Living Standard(s).
+   </td>
+   <td>An explainer describing the  user/developer problem to be solved, including sketching use cases and scenarios. This explainer can exist anywhere, including e.g. a GitHub issue, or a personal repository.
+   </td>
+  </tr>
+  <tr>
+   <td>Stage 1
+<p>
+(Incubation)
+   </td>
+   <td>A comprehensive explainer for the feature, in a standards organization-approved incubation venue such as a <a href="https://www.w3.org/community/groups/">W3C CG</a> or a branch of an existing WHATWG Standard.
+<p>
+Consensus that the WHATWG is interested in exploring solutions in this problem space.
+<p>
+(At least) one implementer interested in doing prototyping work.
+<p>
+Identification of the feature's champion. \
```

Comment:
These stray backslashes confused me. Are they intended to be there? It looks like they were intended to escape newlines in some different document format.

Reply:
Yeesh, yes, the bullets went haywire.  Fixed.